frontend/src/metabase/visualizations/components/settings/ChartSettingFieldsPicker.jsx

```diff
@@ -51,6 +53,11 @@ const ChartSettingFieldsPicker = ({
                         >
                           <ChartSettingFieldPicker
                             {...props}
+                            showColumnSetting={
+                              _.isArray(showColumnSetting)
+                                ? showColumnSetting.includes(index)
```

Comment:
`index` is used pretty far from where it's declared (almost 20 lines apart), do you think giving it a more specific name would help this part be easier to read? e.g.
```suggestion
                                ? showColumnSetting.includes(columnIndex)
```